samples/app-complete-sample/csharp/README.md

```diff
@@ -9,9 +9,10 @@ extensions:
   contentType: samples
   technologies:
   - Tabs
-  - Microsoft Bot Framework
+  - Microsoft Bot Framework v4
   createdDate: 9/22/2017 5:54:09 PM
-description: "Sample that shows how to build a bot for Microsoft Teams in C#."
+  updateDate:  
+description: "Sample that shows how to build a bot for Microsoft Teams in C# with bot framework v4."
```

Comment:
Could you please include few screenshots?

Reply:
Added screenshots of app features